clients/admin-ui/src/pages/systems/configure/[id].tsx

```diff
@@ -38,6 +43,30 @@ const ConfigureSystem: NextPage = () => {
     dispatch(setActiveSystem(system));
   }, [system, dispatch]);
 
+  useEffect(() => {
+    const status = router.query.status;
+
+    if (status) {
+      if (status === "succeeded") {
+        toast(successToastParams(`Integration successfully authorized.`));
+      } else {
+        toast(errorToastParams(`Failed to authorize integration.`));
+      }
+      // create a new url without the status query param
+      let newQuery = { ...router.query };
+      delete newQuery.status;
+      const newUrl = {
+        pathname: router.pathname,
+        query: newQuery,
+      };
+
+      // replace the current history entry
+      router.replace(newUrl, undefined, { shallow: true });
+
+      setInitialTabIndex(3);
```

Comment:
Don't like that this is hard-coded, open to suggestions.

Reply:
Are you referring to `setInitialTabIndex(3);`? If might be helpful to create a constant/enum each tab so instead of using `3` you could use `setInitialTabIndex(INTEGRATION_TAB)`